src/components/ContentWrapper.jsx

```diff
@@ -40,20 +42,44 @@ const ContentWrapper = (props) => {
   const [isAuth, setIsAuth] = useState(false);
   const [collapsed, setCollapsed] = useState(false);
 
-  const { experimentId, experimentData, children } = props;
+  const { routeExperimentId, experimentData, children } = props;
   const router = useRouter();
   const route = router?.route || '';
   const navigateTo = useAppRouter();
 
-  const experiment = useSelector((state) => state?.experiments[experimentId]);
+  const currentExperimentIdRef = useRef(routeExperimentId);
+  const activeProjectUuid = useSelector((state) => state?.projects?.meta?.activeProjectUuid);
+  const activeProject = useSelector((state) => state.projects[activeProjectUuid]);
+  const samples = useSelector((state) => state.samples);
+
+  const activeProjectExperimentID = useSelector((state) => (
+    state?.projects[activeProjectUuid]?.experiments[0]));
+  // if the current selected project experiment is processed
+  // and we are in data-management, we don't have experimentId
```

Comment:
if the experiment is not processed, and we are in `data-management`, don't we also not have experiment id?

Reply:
no, thats the experimentId in the URL that we don't have, this is why I get the one under the activeProject

Reply:
Ooh I see.. Is this comment is about the `useEffect` block below? Can you change this comment to make it clearer? Or maybe delete it if we don't need it,